extensions/amp-video/amp-video.md

```diff
@@ -0,0 +1,40 @@
+<!---
```

Comment:
Do the MD file in the final PR otherwise it will be picked up by `https://www.ampproject.org` automatically which will cause confusion as `<amp-video>` already exists and we are just refactoring it.